maplibre/src/render/tile_view_pattern/mod.rs

```diff
@@ -136,7 +136,7 @@ pub trait HasTile {
                 if self.has_tile(child, world) {
                     output.push(child);
                 } else {
-                    new_children.extend(child.get_children())
+                    new_children.extend(child.get_children().into_iter().flatten())
```

Comment:
```suggestion
                    new_children.extend(child.get_children()?)
```

Reply:
This would be an early return if one child has no children. Is this desired?

Reply:
Ah got it, sorry! :)